packages/client/examples/browser/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "version": "0.0.0",
+  "scripts": {
+    "dev": "vite",
+    "build": "vite build",
+    "serve": "vite preview"
+  },
+  "devDependencies": {
+    "vite": "^2.3.7"
```

Comment:
Do we still need this?

Reply:
yes, the alternative is using skypack which is problematic by https://github.com/web3-storage/ipfs-car/issues/27

I need to book some time next week to try to figure out an work around for that